doc/emacs/macport.texi

```diff
@@ -529,17 +529,28 @@ defaults write org.gnu.Emacs Emacs.fringe.attributeStipple alpha:.5
 @code{scroll-bar-background} frame parameter.  If it is @code{nil},
 which is the default, then the frame background color is used instead.
 Its color also affects the appearance of other GUI parts: scroll bars,
-the title bar, the tool bar, (the tab bar on macOS 10.12 and later),
-and popup menus on OS X 10.10 -- macOS 10.13 become light (or dark) if
-the color is considered light (or dark, respectively).  On macOS 10.14
-and later, the appearance of GUI parts other than scroll bars respect
-the global setting, which is available via
-@code{(mac-application-state)}.  On macOS 10.12 and later, the
-title/tool/tab bars look slightly colored with the value of the
-@code{scroll-bar-background} frame parameter (or the frame background
-color) if the frame is focused and not in fullscreen.
-
-@vindex mac-transparent-titlebar
+the title bar, the tool bar, (the tab bar on macOS 10.12 and later), and
+popup menus on OS X 10.10 -- macOS 10.13 become light (or dark) if the
+color is considered light (or dark, respectively).  On macOS 10.14 and
+later, the appearance of GUI parts other than scroll bars respect the
+global setting, which is available via @code{(mac-application-state)},
+unless @code{mac-appearance} frame parameter is non-@code{nil}.  On
+macOS 10.12 and later, the title/tool/tab bars look slightly colored
+with the value of the @code{scroll-bar-background} frame parameter (or
+the frame background color) if the frame is focused and not in
+fullscreen.
+
+@vindex mac-appearance@r{, a frame parameter}
+If set to @code{dark} draw this frame's window-system window using the
```

Comment:
and toolbar/etc.?

Reply:
Indeed. I thought the ominous `frame's window-system window` covered that aspect. But can change. Will welcome a suggestion though.